tests/s903/test_la_agg.py

```diff
@@ -39,14 +44,37 @@ def test_deduplicate():
 
 
 def test_remove_old_data():
-    this_year = pd.to_datetime("today").year
-    month = pd.to_datetime("today").month
-    last_year = this_year - 1
-    two_ya = this_year - 2
-    three_ya = this_year - 3
-    test_df_1 = pd.DataFrame({"YEAR": [this_year, last_year, two_ya, three_ya]})
-    output_df_1 = process.remove_old_data(test_df_1, 1)
-    if month <= 6:
-        assert len(output_df_1) == 3
-    else:
-        assert len(output_df_1) == 2
\ No newline at end of file
+    latest_data_year = 2022
```

Comment:
It's unnecessary to create the year variables here and call them in later as they're not being reused anywhere else and makes it slightly more difficult to read. I would just have:
`test_df_1 = pd.DataFrame(
    {
        "YEAR": [
                       2022,
                       2021,
                       2020, etc.`

Reply:
Good call! Made the change.